### ISSUE TITLE
tensorflow lite for android doesn't work with gradle version 4.4

### ISSUE THREAD

**Issue by deeppandya** [tensorflow/tensorflow#19275]
I am using mobilenetv2 pre-trained models to inference images. tensorflow lite for android takes 3 times more time to inference with gradle 4.4. if you change the gradle version to 4.1, it works as expected.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by shashishekhar** [comment]
@deeppandya  There was a brief performance regression that slowed down convolutions, the regression has been fixed. Can you try again if you are still seeing the behavior with master.

**Comment by deeppandya** [comment]
@shashishekhar the performance is still the same. with gradle 4.4, it takes around 600-700 ms to inference single frame and with gradle 4.1, it takes only 60-80 ms. the difference is huge. and that's only change i have in the code.

**Comment by shashishekhar** [comment]
@deeppandya is it possible to attach link to apks? Also did you have to make any changes to the gradle file. I am wondering if it is release vs debug binary issue. Are the sizes of apks different?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @deeppandya: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by shashishekhar** [comment]
Closing due to no activity.
